frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2101,6 +2115,10 @@ function ReaderFooter:_updateFooterText(force_repaint, full_repaint)
         return
     end
     local text = self:genFooterText()
+    for dummy, v in ipairs(self.additional_footer_content) do
+        text = v() .. " " .. self:get_separator_symbol() .. " " .. text
+    end
```

Comment:
Can this be done in `genAllFooterText()`, in usual way like other generators?

Reply:
No, it can't directly.
The idea was to import external generators, not just a piece of text, but it is an overengineering definitely.